doc/release_new_version.md

```diff
@@ -38,3 +38,7 @@ Also update build pipelines YAMLS
 Update [AzureDevOpsCli-Released-Version-Check](https://dev.azure.com/ms/azure-devops-cli-extension/_build?definitionId=36) to run from the branch which was used to do the release.
```

Comment:
May be remove these lines?
Since **released version check** was renamed to **release-version** ,
and both text says the same thing.